include/z64player.h

```diff
@@ -317,6 +317,13 @@ typedef enum PlayerDoorType {
     /*  5 */ PLAYER_DOORTYPE_PROXIMITY
 } PlayerDoorType;
 
+// Some player animations are played at this reduced speed, for reasons yet unclear.
+// Perhaps to compress animation data? Or to adjust from 30fps to 20fps?
```

Comment:
Adjusting from 30fps to 20fps isn't it, if it was the case the animations would be played at a faster speed not slower
I'd leave it out, sorry for the misinformation